builtin/logical/transit/backend.go

```diff
@@ -64,8 +64,8 @@ type backend struct {
 }
 
 func (b *backend) invalidate(_ context.Context, key string) {
-	if b.Logger().IsTrace() {
-		b.Logger().Trace("transit: invalidating key", "key", key)
+	if b.Logger().IsDebug() {
+		b.Logger().Debug("transit: invalidating key", "key", key)
```

Comment:
Remove `transit:`

Reply:
Just double-checked `builtin/logical/*` and fixed up those and a couple more I found.